cdap-ui/app/directives/complex-schema/complex-schema.less

```diff
@@ -63,42 +58,39 @@ my-complex-schema {
       width: ~"calc(100% - 75px - 80px)";
 
       &.no-type {
-        width: ~"-moz-calc(100% - 75px)";
-        width: ~"-webkit-calc(100% - 75px)";
-        width: ~"calc(100% - 75px)";
+        width: ~"-moz-calc(100% - 75px - 80px)";
+        width: ~"-webkit-calc(100% - 75px - 80px)";
+        width: ~"calc(100% - 75px - 80px)";
       }
 
       .form-control { padding-left: 5px; }
 
       &.header { padding-left: 4px; }
     }
 
-    .select-wrapper { .select-wrapper(@background-color: transparent, @arrow-color: @cdap-header, @right: 7px); }
+    .select-wrapper {
+      height: 33px;
+      .select-wrapper(@background-color: transparent, @arrow-color: @cdap-header, @right: 7px);
+    }
 
     .tab-header {
-      box-shadow: 0 -7px 6px -6px rgba(1, 0, 0, 0.25), -7px 0 6px -6px rgba(1, 0, 0, 0.25), 7px 0 6px -6px rgba(1, 0, 0, 0.25);
+      border-radius: 4px 4px 0 0;
+      box-shadow: rgba(1, 0, 0, 0.25) 0 0 10px;
+      height: 26px;
       position: relative;
-      &:before,
-      &:after {
-        content: '';
-        display: block;
-        position: absolute;
-        width: 20px;
-        height: 34px;
-        top: 10px;
-      }
-      &:before {
-        box-shadow: -7px 0 6px -6px rgba(1, 0, 0, 0.25);
-        left: 0;
-      }
-      &:after {
-        box-shadow: 7px 0 6px -6px rgba(1, 0, 0, 0.25);
-        right: 0;
-      }
+      z-index: 100;
```

Comment:
@elfenheart this is so that the box shadow applied to the pseudoselector doesn't appear on top of the tab headers